src/Google/Ads/GoogleAds/Lib/V1/LogMessageFormatter.php

```diff
@@ -40,6 +41,26 @@ public function __construct($statusMetadataExtractor = null)
         $this->statusMetadataExtractor = $statusMetadataExtractor ?: new StatusMetadataExtractor();
     }
 
+    /**
+     * Returns the customer ID information of the provided request.
+     *
+     * @param Message $request the request to get its customer ID
+     * @return string the customer ID information
+     */
+    private static function getCustomerIdInfo(Message $request): string
+    {
+        if (method_exists($request, 'getCustomerId')) {
+            return $request->getCustomerId();
+        } elseif (method_exists($request, 'getResourceName')) {
+            $resourceName = $request->getResourceName();
+            $segments = explode('/', $resourceName);
+            if ($segments[0] === 'customers') {
+                return $segments[1];
+            }
+        }
+        return '"No customer ID found in the request"';
```

Comment:
I would change for something like `No customer ID could be extracted from the request`

Reply:
Done.